### ISSUE TITLE
How to get network convergent?

### ISSUE THREAD

**Issue by haithanhp** [SimJeg/FC-DenseNet#33]
Hi @SimJeg ,

Thanks for your great work. I have the following concerns:
1. How many epochs to get the model convergent on camvid data?
2. In the paper, you use exponential decay of -0.995 for learning rate (1e-3). Is that mean you compute lr = 1e-3 * e^(-0.995 * epoch_id) for each epoch?
3. I want to reproduce your results in pytorch. I test on 9 images with visualized grounth truth and predictions below. But after 50 epochs, I did not see better. 

Could you share with the tricks to train model?
Best,
Hai

![epoch_0060](https://user-images.githubusercontent.com/25809722/36766176-c1bb49a4-1c02-11e8-871b-03c65839f31e.jpg)
